llumnix/global_scheduler/global_scheduler.py

```diff
@@ -30,16 +30,19 @@ def __init__(self,
         # instance load and instance info args
         self.load_metric = global_scheduler_config.load_metric
         self.enable_defrag = global_scheduler_config.enable_defrag
+        self.enable_pd_disaggregation = global_scheduler_config.enable_pd_disaggregation
```

Comment:
PDD is the external manifestation of expected_steps = 1.
I thinl that self.enable_pd_disaggregation -> self.expected_steps is better.